internal/ir/xds.go

```diff
@@ -763,6 +763,8 @@ type DNS struct {
 	DNSRefreshRate *metav1.Duration `json:"dnsRefreshRate,omitempty"`
 	// RespectDNSTTL indicates whether the DNS Time-To-Live (TTL) should be respected.
 	RespectDNSTTL *bool `json:"respectDnsTtl,omitempty"`
+	// LookupFamily allows you to configure a preferred DNS lookup policy.
+	DNSLookupFamily *DNSLookupFamily `json:"lookupFamily,omitempty"`
```

Comment:
```suggestion
	LookupFamily *DNSLookupFamily `json:"lookupFamily,omitempty"`
```

Reply:
cc @guydc , if this struct live in DNS struct, it's fine to me to use api struct directly.

Reply:
Yea, sure, we can drop the dns prefix